app/models/payment/braintree_payment_method_token.rb

```diff
@@ -0,0 +1,3 @@
+class Payment::BraintreePaymentMethodToken < ActiveRecord::Base
```

Comment:
You're going to hate this one but I don't think this naming really makes sense. I think it makes a lot more sense to call `Payment::BraintreePaymentMethod.first.token` than it does to call `Payment::BraintreePaymentMethodToken.first. braintree_payment_method_token`. It also will make a lot more sense to have `Payment::GoCardlessPaymentMethod` and `Payment::BraintreePaymentMethod`